cmd/mongodb-healthcheck/healthcheck/util.go

```diff
@@ -0,0 +1,51 @@
+package healthcheck
+
+import (
+	"context"
+	"encoding/json"
+
+	v "github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb/mongo"
+)
+
+func getStatus(ctx context.Context, client mongo.Client) (ReplSetStatus, error) {
```

Comment:
Why don't we have all the mongo client-related functions together as part of the `type Client interface`? I understand that we are not committing to the interface segregation rule by doing that, but that interface is already containing everything (almost in terms of functionality).

Also the response type seems related to the generic mongo model and maybe can be moved to the mongo model file.

```
type ReplSetStatus struct {
...
}

```


This removes the need to have a utils file completely.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1917/commits/ae932d3a61842217cea12236b43b45e25d50f526